website/docs/reference/warehouse-profiles/tidb-profile.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "TiDB Profile"
+---
+
+:::info Vendor-supported plugin
+
+Some core functionality may be limited. If you're interested in contributing, check out the source code for each repository listed below.
+
+:::
+
+## Overview of dbt-tidb
+**Maintained by:** PingCAP      
+**Author:** Xiang Zhang and Qiang Wu
+**Source:** https://github.com/pingcap/dbt-tidb   
+**Core version:** v1.0.0 and newer   
+**dbt Cloud:** Not Supported
+
+The easiest way to install it is to use pip:
+
+```
+    pip install dbt-tidb
+```
+
+## Connecting to TiDB with **dbt-tidb**
+
+#### User / password authentication
+
+Configure your dbt profile for using TiDB:
+
+##### TiDB connection profile
+<File name='profiles.yml'>
+
+```yaml
+dbt-tidb:
+  target: dev
+  outputs:
+    dev:
+      type: tidb
+      server: 127.0.0.1
+      port: 4000
+      schema: database_name
+      username: tidb_username
+      password: tidb_password
+```
+
+</File>
```

Comment:
there's a lot of great information in the repo readme that would benefit from being here!

Reply:
Thanks. We are quite willing to make this page the source of truth.
We have added some key content to the page to help users get more useful information on the page.

Now, dbt-tidb mainly implements dbt configuration without additional functionsality. If we add any new func later, we will add a copy of config pagel.